<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>FeedLol - Preferences</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QListWidget" name="pageSelector" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>88</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="dragDropMode" >
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors" >
      <bool>false</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="flow" >
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="layoutMode" >
      <enum>QListView::Batched</enum>
     </property>
     <property name="spacing" >
      <number>8</number>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
     <item>
      <property name="text" >
       <string>Identity</string>
      </property>
      <property name="icon" >
       <iconset resource="feedlol.qrc" >
        <normaloff>:/icons/user-identity.svg</normaloff>:/icons/user-identity.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Network</string>
      </property>
      <property name="icon" >
       <iconset resource="feedlol.qrc" >
        <normaloff>:/icons/preferences-system-network.svg</normaloff>:/icons/preferences-system-network.svg</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="identity" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>436</width>
        <height>209</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>FriendFeed</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>FriendFeed nickname:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>ffNickname</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="ffNickname" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Remote key (&lt;a href="http://friendfeed.com/remotekey">&lt;span style=" text-decoration: underline; color:#0057ae;">what is this?&lt;/span>&lt;/a>):</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::RichText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="openExternalLinks" >
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
            </property>
            <property name="buddy" >
             <cstring>ffRemoteKey</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="ffRemoteKey" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>91</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="network" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>209</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Network Proxy</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <widget class="QLabel" name="proxyTypeLabel" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Proxy type:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>proxyType</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QComboBox" name="proxyType" >
              <item>
               <property name="text" >
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>HTTP</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>SOCKS</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="proxyHostLabel" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Host:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>proxyHost</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLineEdit" name="proxyHost" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="proxyPortLabel" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Port:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy" >
               <cstring>proxyPort</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="proxyPort" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="minimumSize" >
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum" >
               <number>65535</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" >
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>66</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" >
           <widget class="QCheckBox" name="proxyAuth" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Authentication required</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="proxyAuthUserLabel" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>User name:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy" >
             <cstring>proxyAuthUser</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QLineEdit" name="proxyAuthUser" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="proxyAuthPasswordLabel" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Password:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy" >
               <cstring>proxyAuthPassword</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proxyAuthPassword" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="echoMode" >
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="icon" >
        <iconset resource="feedlol.qrc" >
         <normaloff>:/icons/dialog-ok.svg</normaloff>:/icons/dialog-ok.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="feedlol.qrc" >
         <normaloff>:/icons/dialog-cancel.svg</normaloff>:/icons/dialog-cancel.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pageSelector</tabstop>
  <tabstop>ffNickname</tabstop>
  <tabstop>ffRemoteKey</tabstop>
  <tabstop>proxyType</tabstop>
  <tabstop>proxyHost</tabstop>
  <tabstop>proxyPort</tabstop>
  <tabstop>proxyAuth</tabstop>
  <tabstop>proxyAuthUser</tabstop>
  <tabstop>proxyAuthPassword</tabstop>
 </tabstops>
 <resources>
  <include location="feedlol.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>309</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageSelector</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>160</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>374</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>379</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
